test/support/schemas.ex

```diff
@@ -284,4 +284,15 @@ defmodule OpenApiSpexTest.Schemas do
       oneOf: [Cat, Dog]
     })
   end
+
+  defmodule WithNilProperties do
+    require OpenApiSpex
+
+    OpenApiSpex.schema(%{
+      type: :object,
+      properties: %{
+        meta: nil
```

Comment:
Is this a valid openapi 3 schema?

Reply:
The closest I can see is the `any` type: https://swagger.io/docs/specification/data-models/data-types/#any

Reply:
Hmm good point. I think maybe our project had a `nil` for a property definition, hoping it would be equivalent to an empty `%Schema{}`. Let me investigate this further.